src/parsers/manifest/dash/parse_adaptation_sets.ts

```diff
@@ -348,10 +370,16 @@ export default function parseAdaptationSets(
                                 hasSignLanguageInterpretation(accessibility) ? true :
                                                                                undefined;
 
+      const representations = parseRepresentations(representationsIR,
+                                                   adaptation,
+                                                   adaptationInfos);
```

Comment:
This breaks `unsafelyBaseOnPreviousAdaptation`, no?

Reply:
Good catch